From ac39ba07be177d2f568ad274ba25b67b844257a6 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:06 -0400
Subject: [net] svcrpc: remove unused rq_resused

Message-id: <1403560314-32525-5-git-send-email-bfields@redhat.com>
Patchwork-id: 84445
O-Subject: [Patch RHEL7 004/172] svcrpc: remove unused rq_resused
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

I forgot to remove this in
afc59400d6c65bad66d4ad0b2daf879cbff8e23e "nfsd4: cleanup: replace
rq_resused count by rq_next_page pointer".

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 11d2a1618e377236facdd404113992bde1083914
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/sunrpc/svc.h b/include/linux/sunrpc/svc.h
index 1f0216b..6eecfc2 100644
--- a/include/linux/sunrpc/svc.h
+++ b/include/linux/sunrpc/svc.h
@@ -243,7 +243,6 @@ struct svc_rqst {
  struct xdr_buf  rq_res;
  struct page *  rq_pages[RPCSVC_MAXPAGES];
  struct page *  *rq_respages; /* points into rq_pages */
- int   rq_resused; /* number of pages used for result */
  struct page *  *rq_next_page; /* next reply page to use */
 
  struct kvec  rq_vec[RPCSVC_MAXPAGES]; /* generally useful.. */
-- 
1.7.1